Wikipedia/Localizations/ar.lproj/Localizable.strings

```diff
@@ -375,7 +375,6 @@
 "wikitext-upload-result-unknown" = "غير قادر على تحديد نتيجة تحميل النص";
 "wikitext-upload-save" = "جارٍ الحفظ...";
 "wikitext-upload-save-anonymously-warning" = "سوف تحفظ التعديلات تحت اسم عنوان الأيبي الخاص بجهازك. لو $1 سوف يتوفر لك المزيد من الخصوصية.";
-"cc-by-sa-3.0" = "CC BY-SA 3.0";
```

Comment:
It looked like it would be possible to change this key without having the translatewiki script over-write them, but apparently not.